awscli/customizations/wizard/wizards/configure/main.yml

```diff
@@ -0,0 +1,361 @@
+version: "0.2"
```

Comment:
We should retain the file name `_main.yml`. Mainly, `_main.yml` is a reserved value for wizard command generator so that it allows you to jump into the wizard at the `wizard` command (e.g. `aws configure wizard`). Right now to enter the wizard, we need to specify `aws configure wizard main` which is not what we want.